CHANGELOG.md

```diff
@@ -1,16 +1,76 @@
 # UNPUBLISHED (keep this here and add unpublished changes)
 
+# 6.0.0
+
+## New features
+
+-   [`makeObservable(target, annotations)`](../refguide/observable.md#makeobservable) is now the recommended way to make objects with a fixed shape observable, such as classes.
+-   [`makeAutoObservable(target)`](../refguide/observable.md#makeautoobservable) is will automatically determine the annotations used by `makeObservable`. Methods will be marked as 'autoAction', so that they can be used both from a computed value or as stand alone method.
```

Comment:
```suggestion
-   [`makeAutoObservable(target)`](../refguide/observable.md#makeautoobservable) will automatically determine the annotations used by `makeObservable`. Methods will be marked as 'autoAction', so that they can be used both from a computed value or as standalone method.
```